src/components/day/Day.js

```diff
@@ -414,6 +414,12 @@ export default class DayComponent extends BaseComponent {
     }
   }
 
+  resetValue() {
+    const emptyValue = '00/00/0000';
```

Comment:
I believe it would be better to just implement the ```emptyValue``` setter like so.

```
get emptyValue() {
  return '00/00/0000';
}
```

Reply:
Done 👍

Reply:
Hi @travist , kindly can you take a look again ?